### PR TITLE
Add USB Host keyboard example

### PR DESCRIPTION
**Pull request by kuldeepdhaka**
library code at https://github.com/libopencm3/libopencm3/pull/674

### CODE REVIEW

File: examples/stm32/f1/other/usbh_kbd/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+##
+## This file is part of the libopencm3 project.
+##
+## Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
```
Comment (karlp) [inline examples/stm32/f1/other/usbh_kbd/Makefile]: no, uwe had _nothing_ to do with _this_ file.  Either drop the line altogether, or at least make it correct.

Comment (kuldeepdhaka) [inline examples/stm32/f1/other/usbh_kbd/Makefile]: @karlp removed it.

**Comment by kuldeepdhaka** [comment]
Reason: https://github.com/libopencm3/libopencm3/pull/674 closed

**Comment by ironsteel** [comment]
Anyone interested in continuing this work? Seems that that source branch is deleted from the repo?

**Pull request closed without merging**